lighthouse-core/lib/i18n/i18n.js

```diff
@@ -180,36 +188,40 @@ function _collectAllCustomElementsFromICU(icuElements, seenElementsById = new Ma
  * will apply Lighthouse custom formatting to the values based on the argumentElement
  * format style.
  *
- * @param {Array<import('intl-messageformat-parser').Element>} argumentElements
+ * @param {string} icuMessage
+ * @param {Array<ArgumentElement>} argumentElements
  * @param {Record<string, string | number>} [values]
+ * @return {Record<string, string | number>}
  */
-function _processParsedElements(argumentElements, values = {}) {
-  // Throw an error if a message's value isn't provided
-  argumentElements
-    .filter(el => el.type === 'argumentElement')
-    .forEach(el => {
-      if (el.id && (el.id in values) === false) {
-        throw new Error(`ICU Message contains a value reference ("${el.id}") that wasn't provided`);
-      }
-    });
-
-  // Round all milliseconds to the nearest 10
-  argumentElements
-    .filter(el => el.format && el.format.style === 'milliseconds')
-    // @ts-ignore - el.id is always defined when el.format is defined
-    .forEach(el => (values[el.id] = Math.round(values[el.id] / 10) * 10));
-
-  // Convert all seconds to the correct unit
-  argumentElements
-    .filter(el => el.format && el.format.style === 'seconds' && el.id === 'timeInMs')
-    // @ts-ignore - el.id is always defined when el.format is defined
-    .forEach(el => (values[el.id] = Math.round(values[el.id] / 100) / 10));
-
-  // Replace all the bytes with KB
-  argumentElements
-    .filter(el => el.format && el.format.style === 'bytes')
-    // @ts-ignore - el.id is always defined when el.format is defined
-    .forEach(el => (values[el.id] = values[el.id] / 1024));
+function _processParsedElements(icuMessage, argumentElements, values = {}) {
+  for (const {id, format} of argumentElements) {
+    // Throw an error if a message's value isn't provided
+    if (id && (id in values) === false) {
+      // eslint-disable-next-line max-len
```

Comment:
Lots of ignoring line-length here :cry:

Reply:
summoning @connorjclark and the word-wrap gang! 😈 

![image](https://user-images.githubusercontent.com/2301202/63302616-2d7aef80-c2a3-11e9-9303-803cedb0abba.png)

Reply:
> Lots of ignoring line-length here 😢

I think in the tests there's no value in breaking the string literals up, it just makes the expectations harder to read.

For these two, they're error messages that are only for development time (if a user gets them that means LH will be totally broken), so I don't see much benefit in breaking them in two and multiple lines kind of distracts from the non-error flow of the function, but I also don't feel strongly about it.

Reply:
imo the line length lint rule should ignore long lines that are 80% a single string :sunglasses:

Reply:
> imo the line length lint rule should ignore long lines that are 80% a single string

<img src="https://i.kym-cdn.com/photos/images/original/001/483/348/bdd.jpg" width="100" height="100">

Reply:
broke the strings...for cases of disagreement we should default to our established style guide/lint rules :)